Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '>=3.1.2'
+ruby '3.2.2'
```

Comment:
Please change back ruby version to `ruby '>=3.1.2'` in order to allow seamless integration for every collaborator

Reply:
Great, understood, @stephenkati. 👍